src/routes/spa/spa-router.ts

```diff
@@ -17,16 +17,19 @@ let indexHtmlContent: string = "";
 
 SpaRouter.use(JwtHandlerWithoutQsh);
 SpaRouter.use("/*", async (_, res) => {
+	const { jiraHost } = res.locals;
+	const featureFlags = {
+		ENABLE_5KU_BACKFILL_PAGE: await booleanFlag(BooleanFlags.ENABLE_5KU_BACKFILL_PAGE, jiraHost)
+	};
+
 	if (!indexHtmlContent) {
-		const { jiraHost } = res.locals;
-		const featureFlags = {
-			ENABLE_5KU_BACKFILL_PAGE: await booleanFlag(BooleanFlags.ENABLE_5KU_BACKFILL_PAGE, jiraHost)
-		};
-
-		indexHtmlContent = (await fs.readFile(path.join(process.cwd(), "spa/build/index.html"), "utf-8"))
-			.replace("##SPA_APP_ENV##", envVars.MICROS_ENVTYPE || "")
-			.replace("##SENTRY_SPA_DSN##", envVars.SENTRY_SPA_DSN || "")
-			.replace("\"##FRONTEND_FEATURE_FLAGS##\"", JSON.stringify(featureFlags));
+		indexHtmlContent = await fs.readFile(path.join(process.cwd(), "spa/build/index.html"), "utf-8");
 	}
+
+	indexHtmlContent = indexHtmlContent
```

Comment:
This will override the global `indexHtmlContent`. I'd suggest use a local variable.

Reply:
Yep we are overriding these values every time when the request is called, so that we get the latest value.